_posts/2020-03-02-ios-deploys-super-weird-totally-normal.markdown

```diff
@@ -0,0 +1,113 @@
+---
+layout: epic
+title: "iOS Deploys: Super Weird And Totally Normal"
+date: 2020-03-02
+categories: [ios, deployment, continuous deployment, fastlane]
+author: ash
+---
+
+Software deploys! What a concept. You have some code running somewhere, and you need to get it running somewhere
+else. What could possibly go wrong? While web developers have become accustom to some _really slick_ deploy
+processes, iOS developers have to work within some very different constraints.
+
+Today I want to explore the differences between deploying iOS software and front-end/back-end web software. Some of
+these differences are inherent to how the code gets executed, and some of the differences are incidental to choices
+that Apple has made. These are constraints that iOS developers need to work within. As Artsy has adopted React
+Native over the pats four years, we have had more and more of our web engineering colleagues contributing to our
+iOS app. For these web engineers, getting familiar with the iOS deploy constraints is as important as getting to
+know Xcode and CocoaPods.
+
+<!-- more -->
```

Comment:
What would you think about also adding a section title here, something like "Deploying the Artsy app: a case study"?